### ISSUE TITLE
No shortcut created

### ISSUE THREAD

**Issue by benchmar** [jameshnsears/QuoteUnquote#3]
I installed this app but there is no shortcut to launch it. It is listed in the Android apps list.
Using Android 9 with Color OS6 on Oppo Realme RMX1805

**Comment by jameshnsears** [comment]
Hi - thanks for raising the issue. There is no shortcut - it's a "pure" widget.

I'll raise an issue for possibly adding a launcher icon; in the meantime the following might help you:

https://support.google.com/android/answer/9450271?hl=en#zippy=%2Cadd-or-resize-a-widget

**Comment by benchmar** [comment]
Hi,

Thanks for your prompt reply.

It might be helpful if there was a screen during or post installation about
how to use this app.. most people (me included) expect a homescreen
shortcut to be automatically placed by the installer.

Cheers,

Anuj

**Comment by jameshnsears** [comment]
Hi Anuj - thanks for explaining things. The shortcut is a good idea: though it's not going to be trivial to implement is my guess.

I've created a project board to help track issues raised in GitHub: https://github.com/jameshnsears/QuoteUnquote/projects/1

I'll attempt to code your feature after I have implemented an export feature.

Don't hold your breath; as I'm a bit slow. But you'll be able to track "progress" on that project board & in this issue.

**Comment by jameshnsears** [comment]
App now contains shortcut. Software released; but might take a few days to appear in Google Play.

If you have any other suggestions / spot problems please raise another GitHub issue.

**Comment by benchmar** [comment]
Hi James,

Nice to see that you followed up on my mail/suggestion. It indicates
passion and reminds me of my younger days when I used to tinker around with
hardware & occasionally software.

That was waaay back - my first computer was a ZX Spectrum (not even floppy
drives, we used tape cassettes), then MS-DOS, wINDOWS 3 etc.
I'm sort of a relic now and lost touch with programming (used to do dB apps
with VB frontend for usual payroll, invoicing etc).

I liked your idea of putting together famous quotes and saw a gap which
could be improved upon. Happy to see you take it constructively - always a
good sign.

I'm sure you do well in whatever career you choose.

Best wishes & luck,

Anuj
Mumbai, India

*“The expert in anything was once a beginner"*
